src/utils/dot-env.js

```diff
@@ -19,8 +19,9 @@ const loadDotEnvFiles = async function ({ projectDir }) {
   return response.filter((el) => el.file && el.env)
 }
 
-const tryLoadDotEnvFiles = async ({ projectDir }) => {
-  const dotenvFiles = ['.env', '.env.development']
+const defaultEnvPriority = ['.env', '.env.development', '.env.local', '.env.development.local']
```

Comment:
Can we keep the current defaults to avoid a breaking change in behavior?

Reply:
ok